packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -217,15 +227,21 @@ export const makeAnchorAsset = async (
 };
 harden(makeAnchorAsset);
 
-/** @param {BootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
+/** @typedef {import('./econ-behaviors.js').EconomyBootstrapSpace} EconomyBootstrapSpace */
+
+/** @param {BootstrapSpace & EconomyBootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
+// /** @param {BootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
```

Comment:
Did you mean to keep this commented-out line?

Reply:
nope.